### PR TITLE
change kfctl cmd and help msgs

### PR DESCRIPTION
**Pull request by swiftdiaries**
/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4351)

### CODE REVIEW

**Comment by swiftdiaries** [comment]
Issue for refernce: https://github.com/kubeflow/kubeflow/issues/3518

File: bootstrap/cmd/kfctl/cmd/apply.go [hunk 1]
```diff
@@ -34,9 +34,11 @@ var err error
...
 // applyCmd represents the apply command
 var applyCmd = &cobra.Command{
-	Use:   "apply [all(=default)|k8s|platform]",
-	Short: "Deploy a generated kubeflow application.",
-	Long:  `Deploy a generated kubeflow application.`,
+	Use:   "apply -f ${CONFIG}",
+	Short: "deploys a kubeflow application.",
+	Long: `'kfctl apply' builds and deploys a kubeflow application from a KFDef config.` + "\n" +
+		`To install Kubeflow run -> ` + DeprecatedPrint("kfctl apply -f ${CONFIG}") + "\n" +
+		`For more information, run 'kfctl apply -h' or the docs at www.kubeflow.org.`,
```
Comment (jlewi) [inline bootstrap/cmd/kfctl/cmd/apply.go]: "or read the docs" at

File: bootstrap/cmd/kfctl/cmd/apply.go [hunk 2]
```diff
@@ -90,8 +92,10 @@ func init() {
 	applyCmd.PersistentFlags().StringVarP(&configFilePath, string(kftypes.FILE), "f", "",
 		`Static config file to use. Can be either a local path or a URL.
 For example:
---file=https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_platform_existing.yaml
---file=kfctl_platform_gcp.yaml`)
+export CONFIG=https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_platform_existing.yaml
```
Comment (jlewi) [inline bootstrap/cmd/kfctl/cmd/apply.go]: nit: Can we define a constant for the config file to print out in the command line? I have a feeling people will end up trying whatever we put in the help so we will want to update it over time.

I think we might want to default to something on the v0.7-branch.

File: bootstrap/cmd/kfctl/cmd/generate.go [hunk 3]
```diff
@@ -15,91 +15,26 @@
...
 
-	kftypes "github.com/kubeflow/kubeflow/bootstrap/v3/pkg/apis/apps"
-	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 )
 
-var generateCfg = viper.New()
-
+var generateDeprecationMessage = DeprecatedPrint("'kfctl generate' has been DEPRECATED.") + "\n" +
```
Comment (jlewi) [inline bootstrap/cmd/kfctl/cmd/generate.go]: nit 'kfctl generate' has been replaced by `kfctl build`

File: bootstrap/cmd/kfctl/cmd/generate.go [hunk 4]
```diff
@@ -15,91 +15,26 @@
...
+`Please switch to new semantics.
+To build a KFAPP run -> ` + DeprecatedPrint("kfctl build -f ${CONFIG}") + "\n" +
+`Then to install -> ` + DeprecatedPrint("kfctl apply") + "\n" +
```
Comment (jlewi) [inline bootstrap/cmd/kfctl/cmd/generate.go]: What does the function DeprecatedPrint do? 

kfctl apply -> kfctl apply -f ${CONFIG}

File: bootstrap/cmd/kfctl/cmd/init.go [hunk 5]
```diff
@@ -15,136 +15,28 @@
...
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
+	"github.com/fatih/color"
 )
 
-var initCfg = viper.New()
-
+// DeprecatedPrint sets the color for deprecation warnings
+var DeprecatedPrint = color.New(color.FgYellow).SprintFunc()
+var initDeprecationMessage = DeprecatedPrint("'kfctl init' has been DEPRECATED.") + "\n" +
```
Comment (jlewi) [inline bootstrap/cmd/kfctl/cmd/init.go]: Would it be better to say "removed" rather than DEPRECATED?

**Comment by jlewi** [comment]
Thanks for the quick fix!

File: bootstrap/cmd/kfctl/cmd/apply.go (hunk 1 above)
Comment (swiftdiaries) [reply bootstrap/cmd/kfctl/cmd/apply.go]: done

File: bootstrap/cmd/kfctl/cmd/apply.go (hunk 2 above)
Comment (swiftdiaries) [reply bootstrap/cmd/kfctl/cmd/apply.go]: done

File: bootstrap/cmd/kfctl/cmd/generate.go (hunk 3 above)
Comment (swiftdiaries) [reply bootstrap/cmd/kfctl/cmd/generate.go]: done

File: bootstrap/cmd/kfctl/cmd/init.go (hunk 5 above)
Comment (swiftdiaries) [reply bootstrap/cmd/kfctl/cmd/init.go]: done

File: bootstrap/cmd/kfctl/cmd/generate.go (hunk 4 above)
Comment (swiftdiaries) [reply bootstrap/cmd/kfctl/cmd/generate.go]: It prints out in color to add more "context" to the message.

Screenshot:
Screen Shot 2019-10-19 at 17 48 20

**Pull request merged**